### PR TITLE
caption-this-0109

### PR DESCRIPTION
**Pull request by ElMT2023**
my file has been updated

### CODE REVIEW

**Comment by ElMT2023** [comment]
### 
Closes
fixes
resolves 100

**Comment by ElMT2023** [comment]
reopen the case for review

**Comment by frehiwotberhe** [comment]
Good Job

File: _slides/07-ElMT2023.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ title: "Welcome to our slide deck!"
 ---
 
 Caption here
+salut
```
Comment (frehiwotberhe) [inline _slides/07-ElMT2023.md:7]: Salut
Good job

**Review by frehiwotberhe — COMMENTED** [review]
Nice work

**Comment by ElMT2023** [comment]
This PR updates the copy of my slide because it was out of date
This PR closes / fixes / resolves #1 
## Done
- made a commit
- created a branch
## To Do
- [ ] merge the branch
- [ ] delete the branch afterwards
- [ ] add 3rd task for demo purposes

**Review by Daduc38 — APPROVED** [review]
👍

**Pull request merged**